Game.cpp

```diff
@@ -0,0 +1,95 @@
+// C++ program to implement tic tac toe game 
+#include <iostream> 
+using namespace std; 
+  
+// Function to draw the Tic-Tac-Toe board 
+void drawBoard(char board[3][3]) 
+{ 
+    cout << "-------------\n"; 
+    for (int i = 0; i < 3; i++) { 
+        cout << "| "; 
+        for (int j = 0; j < 3; j++) { 
+            cout << board[i][j] << " | "; 
+        } 
+        cout << "\n-------------\n"; 
+    } 
+} 
+  
+// Function to check for a win 
+bool checkWin(char board[3][3], char player) 
+{ 
+    // Check rows, columns, and diagonals 
+    for (int i = 0; i < 3; i++) { 
+        if (board[i][0] == player && board[i][1] == player 
+            && board[i][2] == player) 
+            return true; 
+        if (board[0][i] == player && board[1][i] == player 
+            && board[2][i] == player) 
+            return true; 
+    } 
+    if (board[0][0] == player && board[1][1] == player 
+        && board[2][2] == player) 
+        return true; 
+    if (board[0][2] == player && board[1][1] == player 
+        && board[2][0] == player) 
+        return true; 
+    return false; 
+} 
+  
+int main() 
+{ 
+    // Initialize the board and players 
+    char board[3][3] = { { ' ', ' ', ' ' }, 
+                         { ' ', ' ', ' ' }, 
+                         { ' ', ' ', ' ' } }; 
+    char player = 'X'; 
+    int row, col; 
+    int turn; // Declare turn here 
+  
+    cout << "Welcome to Tic-Tac-Toe!\n"; 
+  
+    // Game loop 
+    for (turn = 0; turn < 9; turn++) { 
+        // Draw the board 
+        drawBoard(board); 
+  
```

Comment:
In the game loop, instead of using a magic number `9` for the number of turns, consider defining a constant. This will improve code readability and maintainability.